dashboard/app/views/levels/editors/_blockly.html.haml

```diff
@@ -183,38 +183,50 @@
       matchBrackets: true
     });
 
-    var callouts = callout_editor.getValue();
+    var callout_spaces = 0;
+    var updated_callouts = [];
+    updated_callouts.push({});
+
+    fill_form_with_existing_callouts();
+    for(i=0; i<updated_callouts.length; i++) {
+      save_callouts_in_updated_callouts(i);
+    }
+
+    function create_new_callout_space(id) {
+      var callout_html = $("#callout_template").html();
+      callout_html = callout_html.replace(/_\d/g, "_" + ++id)
```

Comment:
I think this will go away if you select by `.callout_2 .localization_key`.  But if not, I prefer `(id + 1)` over `++id` here.  Pre-increment without using `id` again is a little confusing.